.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,12 @@
-* [ ] Put an X in this bracket to confirm you have read the
-  [External Contribution Guidelines](https://github.com/leanprover/lean4/blob/master/doc/contributions.md).
```

Comment:
Why did you remove the checkbox?

Reply:
The whole "Before submitting" section is supposed to be removed, so there's no need to check the checkbox.

I thought it would be better to ask the submitters to remove the "Before submitting" section because of two reasons:

* Less text to read for maintainers
* Less repetitive actions for submitters (e.g. for people who are already familiar with the contribution guidelines)